workspaces/topology/plugins/topology/report.api.md

```diff
@@ -19,5 +25,319 @@ export const topologyPlugin: BackstagePlugin<
+    readonly 'common.enable': string;
+    readonly 'common.disable': string;
+    readonly 'common.off': string;
+    readonly 'common.true': string;
+    readonly 'common.false': string;
+    readonly 'common.online': string;
+    readonly 'common.offline': string;
+    readonly 'common.connected': string;
+    readonly 'common.disconnected': string;
+    readonly 'common.available': string;
+    readonly 'common.unavailable': string;
+    readonly 'common.healthy': string;
+    readonly 'common.unhealthy': string;
+    readonly 'common.normal': string;
+    readonly 'common.critical': string;
+    readonly 'common.high': string;
+    readonly 'common.medium': string;
+    readonly 'common.low': string;
+    readonly 'common.info': string;
+    readonly 'common.debug': string;
+    readonly 'common.trace': string;
+    readonly 'common.today': string;
+    readonly 'common.yesterday': string;
+    readonly 'common.thisWeek': string;
+    readonly 'common.lastWeek': string;
+    readonly 'common.thisMonth': string;
+    readonly 'common.lastMonth': string;
+    readonly 'common.thisYear': string;
+    readonly 'common.lastYear': string;
+    readonly 'common.never': string;
+    readonly 'common.always': string;
+    readonly 'common.sometimes': string;
+    readonly 'common.often': string;
+    readonly 'common.rarely': string;
+    readonly 'common.frequently': string;
+    readonly 'common.occasionally': string;
+    readonly 'common.daily': string;
+    readonly 'common.weekly': string;
+    readonly 'common.monthly': string;
+    readonly 'common.yearly': string;
+    readonly 'common.hourly': string;
+    readonly 'common.minutely': string;
+    readonly 'common.weeks': string;
+    readonly 'common.months': string;
+    readonly 'common.years': string;
+    readonly 'bootOrder.summary': string;
+    readonly 'bootOrder.emptySummary': string;
+    readonly 'bootOrder.disk': string;
+    readonly 'bootOrder.network': string;
+    readonly 'bootOrder.cdrom': string;
+    readonly 'vm.status.starting': string;
+    readonly 'vm.status.stopping': string;
+    readonly 'vm.status.stopped': string;
+    readonly 'vm.status.paused': string;
+    readonly 'vm.status.migrating': string;
+    readonly 'vm.status.provisioning': string;
+    readonly 'vm.status.errorUnschedulable': string;
+    readonly 'vm.status.errorImagePull': string;
+    readonly 'vm.status.imageNotReady': string;
+    readonly 'vm.status.waitingForVolumeBinding': string;
+  }
+>;
+
+// @public
+export const topologyTranslations: TranslationResource<'plugin.topology'>;
+
+// @public
+export const Trans: ({
+  message,
+  params,
+  components,
+}: TransProps<Messages>) => any;
+
+// @public
+export interface TransProps<
+  TMessages extends {
+    [key in string]: string;
+  },
+> {
+  // (undocumented)
+  components?: Record<string, ComponentType<any>>;
+  // (undocumented)
+  message: keyof TMessages;
+  // (undocumented)
+  params?: any;
+}
```

Comment:
I didn't looked into the details of this component, but I don't expect that we need to export it from the npm package.